Sources/TuistDependencies/DependenciesGraph/DependenciesGraphController.swift

```diff
@@ -69,10 +75,24 @@ public final class DependenciesGraphController: DependenciesGraphControlling {
     }
 
     public func load(at path: AbsolutePath) throws -> TuistGraph.DependenciesGraph {
-        let graphPath = graphPath(at: path)
-        guard FileHandler.shared.exists(graphPath) else {
-            return .none
+        var graphPath = graphPath(at: path)
+
+        if !FileHandler.shared.exists(graphPath) {
+            // If the current directory does not have a dependency graph available
+            // look at the root of the complete project
+            
+            guard let rootDirectory = self.rootDirectoryLocator.locate(from: path) else {
+                return .none
+            }
+            let rootGraphPath = self.graphPath(at: rootDirectory)
+            
+            guard FileHandler.shared.exists(rootGraphPath) else {
+                return .none
+            }
+            
+            graphPath = rootGraphPath
         }
```

Comment:
This would mean that it is not supported that sub projects have a `Tuist/Dependencies.swift`, right ? If this is the case you are right I think - we would always try to find the `Dependencies.swift` file at the root

Reply:
If a subproject has a `Tuist/` folder, then the `rootDirectoryLocator.locate` returns the `Tuist` folder itself, isn't it?

Reply:
You are right, then `rootDirectoryLocator.locate` will immediately return the `Tuist` directory of that sub project